src/index.html

```diff
@@ -11,7 +11,169 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="personal_info">
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:<input
+          id="surname"
+          type="text"
+          name="surname"
+          required
+          autocomplete="off">
```

Comment:
Виправ тут і в інших місцях
```suggestion
          Surname:
          <input
            id="surname"
            type="text"
            name="surname"
            required
            autocomplete="off"
          >
```